src/app/modules/group/components/group-manager-list/group-manager-list.component.ts

```diff
@@ -75,29 +79,73 @@ export class GroupManagerListComponent implements OnChanges, OnDestroy {
     this.selection = managers;
   }
 
-  onRemove(): void {
-    if (this.selection.length === 0 || !this.groupData) {
+  onRemove(event: Event): void {
+    if (this.selection.length === 0) {
       return;
     }
 
+    const currentUserId = this.userService.session$.getValue()?.user.groupId;
+
+    if (!currentUserId) {
+      throw new Error('Unexpected: Missed current user ID');
+    }
+
+    if (this.selection.some(manager => manager.id === currentUserId)) {
+      this.confirmationService.confirm({
+        target: event.target || undefined,
+        key: 'commonPopup',
+        icon: 'pi pi-exclamation-triangle',
+        message: $localize`Are you sure to remove yourself from the managers of this group. You may lose manager access and 
+          not be able to restore it.`,
+        acceptLabel: $localize`Yes, remove me from the group managers`,
+        acceptButtonStyleClass: 'p-button-danger',
+        acceptIcon: 'fa fa-check',
+        rejectLabel: $localize`No`,
+        accept: () => this.remove(),
+      });
+
+      return;
+    }
+
+    this.remove();
+  }
+
+  remove(): void {
+    if (!this.groupData) {
+      throw new Error('Unexpected: Missed groupData');
+    }
+
+    const currentUserId = this.userService.session$.getValue()?.user.groupId;
+
+    if (!currentUserId) {
+      throw new Error('Unexpected: Missed current user ID');
+    }
+
+    if (this.selection.some(manager => manager.id === currentUserId)) {
+      const foundIndex = this.selection.findIndex(manager => manager.id === currentUserId);
+      const currentUser = this.selection.splice(foundIndex, 1);
+      this.selection = [ ...this.selection, ...currentUser ];
+    }
+
     const groupId = this.groupData.group.id;
 
     this.removalInProgress = true;
 
-    forkJoin(
-      this.selection.map(manager => this.removeGroupManagerService.remove(groupId, manager.id))
-    ).subscribe({
-      next: () => {
-        this.removalInProgress = false;
-        this.feedbackService.success($localize`Selected managers have been removed.`);
-        this.selection = [];
-        this.reloadData();
-      },
-      error: () => {
-        this.removalInProgress = false;
-        this.actionFeedbackService.unexpectedError();
-      }
-    });
+    forkJoin([
+      concat(...this.selection.map(manager => this.removeGroupManagerService.remove(groupId, manager.id))),
+    ])
```

Comment:
the `forkJoin([ concat(...array)])` is a bit hard to get
Isn't there a simpler way ? Like:
```ts
concat(...this.selection.map(…)).subscribe({
  complete /* instead of 'next' */: () => {…},
  error: () => {…},
})
```